app/views/categories/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<header>
```

Comment:
- [ ] When I try to save a category with no URL or invalid URL, I get the following error. Please make sure you have custom error for this. 

![image](https://user-images.githubusercontent.com/58504655/183249658-ca8b1071-a88d-4a45-ba4a-2b7901038163.png)

Reply:
Hello @arslanbisharat, I have just tried the same case and got no error. 
<img width="751" alt="Screenshot 2022-08-06 at 14 19 55" src="https://user-images.githubusercontent.com/17196048/183250972-81778b39-3522-4c1d-b0d5-234eb679e575.png">